src/pages/my/Profile.jsx

```diff
@@ -10,6 +10,65 @@ const MyProfile = () => (
       <div><Link to="/my/orders">My Orders Index</Link></div>
       <div><Link to="/admin/orders">My Admin Orders</Link></div>
     </header>
+    <body>
+      <div className="container">
```

Comment:
Refactor a bit to add Formik and Yup validation? We will need the validation when we retrieve the data from the table and when we update the address from Delivery Details